openai/operations.py

```diff
@@ -142,7 +144,7 @@ def check(config):
         list_models(config, {})
         return True
     except Exception as err:
-        logger.error('{0}'.format(err))
+        logger.exception('{0}'.format(err))
```

Comment:
not required. in the execute function in operations.py, we are already logging the same trace

Reply:
In connectors.py we logged stacktrace only in execute, provided for check_healt